vignettes/download_jira_data.Rmd

```diff
@@ -35,7 +41,7 @@ As usual, the first step is to load the project configuration file.
 # Project Configuration File
 
 ```{r}
-conf <- yaml::read_yaml("../conf/geronimo.yml")
+conf <- yaml::read_yaml("../conf/thrift.yml")
```

Comment:
Please keep the original geronimo config, since the notebook discusses the properties of the data associated to it.